### PR TITLE
Feat/4307 improvements w.r.t responsiveness for the emoji reactions

### PR DESCRIPTION
**Pull request by raphj**
#4307

## Checkpoints

**For developers**

- [X] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [X] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [X] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [X] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_lib/src/component/AppContent/AppContentRightMenu.jsx [hunk 1]
```diff
@@ -8,22 +8,20 @@ import ArchiveDeleteContent from '../OptionComponent/ArchiveDeleteContent.jsx'
...
+    />
+
     {props.loggedUser.userRoleIdInWorkspace >= ROLE.contributor.id && (
-      <>
-        <EmojiReactions
-          apiUrl={props.apiUrl}
-          loggedUserId={props.loggedUser.userId}
-          contentId={props.content.content_id}
-          workspaceId={props.content.workspace_id}
-        />
```
Comment (gbonaspetti) [inline frontend_lib/src/component/AppContent/AppContentRightMenu.jsx:18]: A reader can react?

Comment (gbonaspetti) [reply frontend_lib/src/component/AppContent/AppContentRightMenu.jsx:18]: I see, you put in readOnly

Comment (raphj) [reply frontend_lib/src/component/AppContent/AppContentRightMenu.jsx:18]: they can see the reactions

**Review by gbonaspetti — APPROVED** [review]

**Comment by grignards** [comment]
Tested on branch.

**Pull request merged**